src/common/meta/src/key/flow/flow_info.rs

```diff
@@ -215,6 +215,36 @@ impl FlowInfoManager {
             TxnOpGetResponseSet::decode_with(TxnOpGetResponseSet::filter(key)),
         ))
     }
+
+    /// Builds a update flow transaction.
+    /// It is expected that the `__flow/info/{flow_id}` IS ALREADY occupied and equal to `prev_flow_value`,
+    /// but the new value can be the same, so to allow replace operation to happen even when the value is the same.
+    /// Otherwise, the transaction will retrieve existing value and fail.
+    pub(crate) fn build_update_txn(
+        &self,
+        flow_id: FlowId,
```

Comment:
I think we can use the FlowId in the `FlowInfoValue`?

Reply:
> I think we can use the FlowId in the `FlowInfoValue`?

There is no FlowId in `FlowInfoValue`?